x-pack/plugins/reporting/public/share_context_menu/register_csv_reporting.tsx

```diff
@@ -8,7 +8,7 @@
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 
-import { CSV_JOB_TYPE } from '@kbn/reporting-export-types-csv-common';
+import { CSV_JOB_TYPE, CSV_JOB_TYPE_V2 } from '@kbn/reporting-export-types-csv-common';
```

Comment:
I would just like to give a head's up that this file might be changing quite a bit in https://github.com/elastic/kibana/pull/171253/files

Reply:
😢